Packages/io.chainsafe.web3-unity.web3auth/Runtime/WalletGUI/Scripts/Web3AuthWalletGUITxManager.cs

```diff
@@ -65,9 +65,9 @@ private void Awake()
     /// <summary>
     /// Populates the incoming transaction display.
     /// </summary>
-    private void OnTransactionRequested(TransactionRequested transactionRequested)
+    private void OnTransactionRequested((string id, TransactionRequest request) args)
```

Comment:
Why arent these two params? That seems weird. Then you can make them a tuple when you add them to the queue easily.

Reply:
I did it so I can easily subscribe and unsubscribe like this
![image](https://github.com/ChainSafe/web3.unity/assets/36323674/d5c9fa3e-acaa-4e33-b117-bc7cb9cb6a20)

Reply:
Think that would be a tad cleaner.

Reply:
done